packages/lit-dev-content/samples/properties/attributereflect/my-element.ts

```diff
@@ -1,24 +1,40 @@
-import {LitElement, html, css} from 'lit';
+import {LitElement, html, css, PropertyDeclaration} from 'lit';
 import {customElement, property} from 'lit/decorators.js';
 
 @customElement('my-element')
 class MyElement extends LitElement {
   @property({type: Boolean, reflect: true})
   active: boolean = false;
 
+  @property({type: String, reflect: true, defaultValue: 'normal'} as PropertyDeclaration)
```

Comment:
Is this cast only needed until https://github.com/lit/lit/pull/4895 is released?